make/tests

```diff
@@ -91,14 +81,15 @@ test-headers: $(HEADER_TESTS)
 ##
 test/prob/generate_tests$(EXE) : test/prob/generate_tests.cpp
 	@mkdir -p $(dir $@)
-	$(LINK.cc) -O$(O_STANC) $< $(OUTPUT_OPTION)
```

Comment:
It does, but in the Stan repo. We could remove it from here, but we should hold off until the default is added to Stan first. (Just so we don't leave it undefined.)